src/PostgREST/ApiRequest.hs

```diff
@@ -92,6 +94,10 @@ data ApiRequest = ApiRequest {
   , iCanonicalQS :: ByteString
   -- | JSON Web Token
   , iJWT :: Text
+  -- | HTTP request headers
+  , iHeaders :: [(Text, Text)]
+  -- | Request Cookies
+  , iCookies :: Maybe [(Text, Text)]
```

Comment:
No meaning, it was kind of an effect of the return type of the function used. Probably [] will be better